cellrank/tools/estimators/_base_estimator.py

```diff
@@ -370,15 +372,15 @@ def compute_absorption_probabilities(
             metastable_states_
         )
 
-        # create Q (restriction transient-transient), S (restriction transient-recurrent) and I (Q-sized identity)
+        # create Q (restriction transient-transient), S (restriction transient-recurrent)
         q = t[trans_indices, :][:, trans_indices]
         s = t[trans_indices, :][:, rec_indices]
 
         if check_irred:
             if self._is_irreducible is None:
                 self.compute_partition()
             if not self._is_irreducible:
-                logg.warning("The transition matrix is not irreducible. ")
+                logg.warning("The transition matrix is not irreducible")
```

Comment:
I guess this is really picky, but further up, we terminated warning messages with a point. Shall we do the same here?